index.html

```diff
@@ -1,13 +1,64 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="./style.css">
-    <title>Document</title>
-</head>
-<body>
-    
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
```

Comment:
- [ ] Kindly consider using the correct path to link your `style.css` to your HTML file.
- [ ] Also you may consider fixing your style issues as you can notice how your page is not presentable, and also the error in the console in the `bubble.js`
![image](https://user-images.githubusercontent.com/97100091/205635499-6c8c87a5-cbc0-4918-b187-001b26482265.png)

Reply:
@christianbiring1 Sir, I've found a solution for this error at https://stackoverflow.com/questions/74281137/companion-bubble-js-in-the-console this is a bug from the loom chrome extension .

Reply:
This change has been done well